priya/javascript/Assignment6/JsAssign6Q4.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Array</title>
+    </head>
+    <body>
+        <h4>Enter 5 numbers here</h4>
+        Number 1: <input type="text" id="num1" placeholder="Enter first number "> <br>
+        <br>
+        Number 2: <input type="text" id="num2" placeholder="Enter second number "> <br>
+        <br>
+        Number 3: <input type="text" id="num3" placeholder="Enter third number "> <br>
+        <br>
+        Number 4: <input type="text" id="num4" placeholder="Enter fourth number "> <br>
+        <br>
+        Number 5: <input type="text" id="num5" placeholder="Enter fifth number "> <br>
+        <br>
+        <button onclick="MyArray2()">Enter</button><br>
+        <br>
+        <button id="null">cancel</button>
+        <br>
+        <br>
+        <script language="javascript">
+            function MyArray2(){
+                var b,a,c,d,e,fact, arr;
+                cancel = null;
+                //for input names
+                a= parseInt(document.getElementById("num1").value);
```

Comment:
there should be always a space before and after operator so make this everywhere except in html tags